chart/README.md

```diff
@@ -0,0 +1,102 @@
+# aws-event-sources
+
+Installs Triggermesh AWS event sources controller for knative. The following event sources are currently supported by the controller
```

Comment:
it is not just for knative. we could actually send the events to a regular kubernetes service

Reply:
okay. So I should also remove Knative from the pre-requisites right?

Reply:
we should check with @antoineco I am not sure if the sources pods are running via ksvc themselves. I dont' think so but..

Reply:
SNS does